librz/core/tui/panels.c

```diff
@@ -4238,7 +4245,7 @@ void __init_menu_saved_layout(void *_core, const char *parent) {
 void __init_menu_color_settings_layout(void *_core, const char *parent) {
 	RzCore *core = (RzCore *)_core;
 	const char *color = core->cons->context->pal.graph_box2;
-	char *now = rz_core_cmd_str(core, "eco.");
+	char *now = rz_core_theme_get(core);
```

Comment:
`const char*`

Reply:
can't do `const char*` because there's a split below.
https://github.com/rizinorg/rizin/blob/fb71f368596879eb1664c5c2ac04c0613f425a0f/librz/core/tui/panels.c#L4242

Reply:
I'll make a copy of string and pass it to this split.

Reply:
I changed this, just forgot to free the duplicated string in this commit, I'll push the change in next commit, marking this as resolved.